### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][Jaws/NVDA][MAS4.2.1][IntelliJ][Wipro]:Jaws and NVDA are not announcing the listitems present under comboxes in "Run on web App for Containers" dialog

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1466]
**Repro Steps:**

1. Open and Start IntelliJ.
2. Create a new java project.
3. Tab to and select Project ->Azure->Run on  web app for containers
4. "Run on web app for containers" dialog will open. 
5. In "Run on web app for containers" dialog, tab to and select  "web app for container" button.->create new radio button
6. Tab to Container Registry combo box
    Subscription combo box 
   "use Existing" combo box under resource group expand button.
   "Location and Pricing Tier" combo boxes under "App Service Plan" expand button.
  "Java Version and Artifact" combo boxes under "Java" expand button, and use alt+down arrow to expand the combo box
7. Observe the behaviour of Jaws and NVDA

**Actual Result:**
List items under combo boxes are not announced by Jaws and NVDA

**Jaws behaviour:**
Combo box closed
blank
1 of 24
To change the selection use the arrow keys
Open list Box
open
blank
blank
blank

**NVDA Behaviour:**
combo box  collapsed
list
list item  selected  1 of 23
list item  selected  2 of 23
list item  selected  3 of 23

**Expected Result:**
List items under combobox should be announced by Jaws and NVDA

Jaws should announce as
Use Existing Combo box closed
1 of 23
To change the selection use the arrow keys
Open list Box
open
Terraform TestRG list item  selected  1 of 23
abode RG list item  selected  2 of 23
anu-rg-eh-np list item  selected  3 of 23

NVDA should announce as
Use Existing combo box  collapsed
list
Terraform TestRG list item  selected  1 of 23
abode RG list item  selected  2 of 23
anu-rg-eh-np list item  selected  3 of 23

**Platform:**
IntelliJ IDEA 2017.3.2 (Community edition)
Build# IC-173.4127.27
JRE:1.8.0_152-release-1024-b8amd64
JVM: OpenJDK 64 Bit Server VM by Jetbrains.
OSBuild Version: Version 1709(Os Build 16299.192)
![run on we app-combo box-list items](https://user-images.githubusercontent.com/22956130/36102334-05dd18ba-1032-11e8-902c-cc8a96f52490.jpg)
![14cd8249](https://user-images.githubusercontent.com/22956130/36102339-09935d34-1032-11e8-96a5-8a0717dda133.jpg)
![run on web app for containers-listitems-explorer](https://user-images.githubusercontent.com/22956130/36102341-0c610016-1032-11e8-95f0-2e20e25cad63.jpg)

**Comment by Eskibear** [comment]
Cannot reproduce it.

According to the practice, NVDA cannot read content of any dialog in IntelliJ IDEA 2017.3.4 Community.
A ticket is created to track the issue.

Ref:
https://youtrack.jetbrains.com/issue/IDEA-191325

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749926
